--- origsrc/gpgme-1.1.8/src/Makefile.am	2008-12-03 07:36:07.000000000 -0600
+++ src/gpgme-1.1.8/src/Makefile.am	2010-01-12 03:45:23.583958100 -0600
@@ -190,7 +190,7 @@ libgpgme_la_DEPENDENCIES = $(assuan_libo
 libgpgme_la_LIBADD = $(assuan_libobjs) @LTLIBOBJS@ \
 	@GPG_ERROR_LIBS@ @NETLIBS@
 
-libgpgme_pthread_la_LDFLAGS = $(libgpgme_version_script_cmd) -version-info \
+libgpgme_pthread_la_LDFLAGS = $(no_undefined) $(libgpgme_version_script_cmd) -version-info \
 	@LIBGPGME_LT_CURRENT@:@LIBGPGME_LT_REVISION@:@LIBGPGME_LT_AGE@
 libgpgme_pthread_la_DEPENDENCIES = $(assuan_libobjs) \
 	@LTLIBOBJS@ $(srcdir)/libgpgme.vers
